### ISSUE TITLE
NTR: Vitamin B12 low in maternal blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8372]
New term request Vitamin B12 low in maternal blood (elthat_131214185245): Mother; cobalamin  UMLS: 5439112

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000518
- label: Low maternal vitamin B12 concentration
- parents: [Abnormal maternal serum screening (HP:0011436)](https://hpo.jax.org/app/browse/term/HP:0011436)
- pmid: 
- def: The concentration of vitamin B12 in the mother's blood during pregnancy is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8372
- synonyms:
